compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -1141,9 +1232,12 @@ class Definitions {
     () => ScalaPackageVal.termRef
   )
 
-  val PredefImportFns: List[() => TermRef] = List[() => TermRef](
+  lazy val PredefImportFns: List[() => TermRef] = List[() => TermRef](
     () => ScalaPredefModuleRef,
-    () => DottyPredefModuleRef
+    () => DottyPredefModuleRef,
+    // TODO(abeln): is this in the right place?
+    // And is it ok to import this unconditionally?
+    () => ctx.requiredModuleRef("scala.ExplicitNulls")
```

Comment:
I think it is the right place.

Once we use `explicitNulls` in dotty-library, we may have a problem with bootstrapping `dotty-library`. /cc: @smarter